docs/how-to-guides/how-to-setup-triggers.md

```diff
@@ -0,0 +1,101 @@
+---
+id: how-to-setup-triggers
+title: How to setup triggers
+sidebar_label: Triggers
```

Comment:
Set up triggers

We agreed that how-to guides should have an action/verb in their name, so they kind of differentiate from the reference guide where titles/labels are: Triggers, Streams, Query Modules... so everything about them
And how to guides kind of indicate what they are going to show you to do specifically with triggers